src/app/static/scripts/run-dependencies-for-integration-tests.sh

```diff
@@ -5,9 +5,10 @@ HERE=$(readlink -f "$(dirname $0)")
 NETWORK=hint_nw
 HINT=hint
 $HERE/../../../../scripts/run-dependencies.sh
-HINT_VERSION=mrc-433_integrationtests
+HINT_VERSION=mrc-417_backend
```

Comment:
this is kind of annoyingly circular again, not sure how to handle this properly

Reply:
Should the hint version always be the current branch? No, because that only works if the image exists, and using master is fine, if there's only front end changes I guess. Hmm... It could push the current branch?

Reply:
We can build it and not push it :+1: